app/views/course/assessment/submission/submissions/_submissions.html.slim

```diff
@@ -0,0 +1,12 @@
+table.table.submissions
+  thead
+    tr
+      th = t('.student_name')
+      th = t('.status')
+      th = t('.grade')
+      th = t('.experience_points')
+      th
+  tbody
+    - course_users.each do |course_user|
```

Comment:
Was considering that, then it would be something like this:
```ruby
# _submissions.html.slim
  tbody
    = render partial: 'submission', collection: course_users, locals: { submissions: submissions }

# _submission.html.slim
- submission = submissions.find { |s| s.creator_id == course_user.user_id }
- if submission
  # other code
```

I can do this, but I can reuse the `submission` partial for _pending submissions_. I'm not sure what's the best way to split the partials yet though.